packages/block-library/src/embed/edit.js

```diff
@@ -193,7 +194,9 @@ const EmbedEdit = ( props ) => {
 		native: title,
 	} );
 	// No preview, or we can't embed the current URL, or we've clicked the edit button.
-	const showEmbedPlaceholder = ! preview || cannotEmbed || isEditingURL;
+	//const showEmbedPlaceholder = ! preview || cannotEmbed || isEditingURL;
+	const showEmbedPlaceholder = false;
```

Comment:
We have to hide the embed placeholder to test the PR but I'd revert this change once it's approved.